Client/Frontend/Browser/BrowserViewController/BrowserViewController+TabToolbarDelegate.swift

```diff
@@ -54,6 +54,12 @@ extension BrowserViewController: TabToolbarDelegate, PhotonActionSheetProtocol {
             showLibrary()
         }
     }
+    
+    func tabToolbarDidPresssAddNewTab(_ tabToolbar: TabToolbarProtocol, button: UIButton) {
```

Comment:
Added new method to handle when ➕ is tapped we open a new tab and focus on the URL Bar so user can easily search.

Reply:
To many s in `Presss`